pkgs/tools/networking/ssmtp/default.nix

```diff
@@ -10,6 +10,8 @@ stdenv.mkDerivation {
     sha256 = "0dps8s87ag4g3jr6dk88hs9zl46h3790marc5c2qw7l71k4pvhr2";
   };
 
+  patches = [ ./ssmtp_support_AuthPassFile_parameter.patch ];
```

Comment:
Make sense. I just added myself. I will also try to upstream this patch to Debian.